cloudinit/config/modules.py

```diff
@@ -186,6 +187,9 @@ def _fixup_modules(self, raw_mods) -> List[ModuleDetails]:
                 continue
             mod = importer.import_module(mod_locs[0])
             validate_module(mod, raw_name)
+            if freq is None:
```

Comment:
Whoops...that was a pretty big miss on my part :grimacing:

Reply:
My fault in reviewing :)